[BTAT-8019] Upgrade to Scala 2.12 and SBT 1.3

JBlacklawsTen10 opened the pull request.

daparkes commented:
Sorry I didn't notice before but code coverage is slightly below the minimum - here are the lines to add to your excludedPackages in build.sbt so you can just copy+paste:
    "com.kenshoo.play.metrics.*",
    "controllers.javascript.*",

Also, you can solve the "insecure HTTP request is deprecated" warning by changing the Typesafe resolver in plugins.sbt to https:// at the beginning instead of http:// (I would have said this earlier but I only just found out myself from Deng)

Are you able to make those PRs in build-jobs and service-manager-config?

JBlacklawsTen10 commented:
Updated service-manager-config: https://github.com/hmrc/service-manager-config/pull/2430
Updated build-jobs: https://github.com/hmrc/build-jobs/pull/6285
